modules/extensions-support/src/test/java/org/mule/module/extension/HeisenbergExtension.java

```diff
@@ -123,14 +125,26 @@
     @Parameter(alias = "finalHealth")
     private HealthStatus endingHealth;
 
-    @Parameter(expressionSupport = REQUIRED)
+    @Parameter
+    @Expression(expressionSupport = REQUIRED)
     @Optional
     private String labAddress;
 
-    @Parameter(expressionSupport = NOT_SUPPORTED)
+    @Parameter
+    @Expression(expressionSupport = NOT_SUPPORTED)
     @Optional
     private String firstEndevour;
 
+    @Parameter
+    @Optional(defaultValue = "#[payload]")
+    @Expression(expressionSupport = LITERAL)
+    private String literalExpressionWithDefault;
+
+    @Parameter
+    @Optional
+    @Expression(expressionSupport = LITERAL)
```

Comment:
once the expressionSupport field has been renamed to value, you don't need the = anymore